node/full.go

```diff
@@ -0,0 +1,36 @@
+package node
+
+import (
+	"context"
+
+	"go.uber.org/fx"
+
+	"github.com/celestiaorg/celestia-node/node/config"
+	"github.com/celestiaorg/celestia-node/node/p2p"
+)
+
+// NewFull creates and runs a new ready-to-go Full Node.
+// To gracefully stop it the Stop method must be used.
+func NewFull(ctx context.Context, cfg *config.Config) (*Node, error) {
```

Comment:
I disagree. Typical writing for this would be outside of the package, so it would like `node.NewFull` and that's verbose enough. Its common Golang practice not to repeat the work from the package, e.g. `big.NewInt` and not `big.NewBigInt`, etc